queries/languages.ts

```diff
@@ -23,29 +28,77 @@ const query = {
             'data.imageData.id.sourceIdentifier.value^1000.0',
             'data.imageData.id.otherIdentifiers.value^1000.0'
           ],
-          type: 'best_fields',
-          analyzer: 'whitespace_analyzer',
-          operator: 'Or'
+          operator: 'Or',
+          query: '{{query}}',
+          type: 'best_fields'
         }
       },
       {
-        multi_match: {
-          query: '{{query}}',
-          fields: [
-            'data.title^100.0',
-            'data.title.english^100.0',
-            'data.title.shingles^100.0',
-            'data.alternativeTitles^100.0',
-            'data.lettering'
-          ],
-          type: 'best_fields',
-          fuzziness: 'AUTO',
-          operator: 'And'
+        dis_max: {
+          queries: [
+            {
+              bool: {
+                _name: 'title prefix',
+                boost: 1000.0,
+                must: [
+                  {
+                    prefix: {
+                      'data.title.keyword': {
+                        value: '{{query}}'
+                      }
+                    }
+                  },
+                  {
+                    match_phrase: {
+                      'data.title': {
+                        query: '{{query}}'
+                      }
+                    }
+                  }
+                ]
+              }
+            },
+            {
+              multi_match: {
+                _name: 'title exact spellings',
+                fields: [
+                  'data.title^100.0',
+                  'data.title.english^100.0',
+                  'data.title.shingles^100.0',
+                  'data.alternativeTitles^100.0'
+                ],
+                operator: 'And',
+                query: '{{query}}',
+                type: 'best_fields'
+              }
+            },
+            {
+              multi_match: {
+                _name: 'title alternative spellings',
+                fields: [
+                  'data.title^80.0',
+                  'data.title.english^80.0',
+                  'data.title.shingles^80.0',
+                  'data.alternativeTitles^80.0'
+                ],
+                fuzziness: 'AUTO',
+                operator: 'And',
+                query: '{{query}}',
+                type: 'best_fields'
+              }
+            },
+            {
+              dis_max: {
+                _name: 'non-english titles',
+                queries: languageQueries
```

Comment:
Did we try use the multi-match query here? As far as I can tell it uses the same analyser that the data was indexed with. 

I think the [`best_filelds`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#type-best-fields) should deal with this, and make the query more succinct.

Reply:
looks like the results are the same for all of the tests, though the scores are very slightly different. haven't figured out exactly where the difference is coming from, but i'm fine with using that implementation if you're happier with it